tests/parametric/test_headers_tracecontext.py

```diff
@@ -807,6 +814,201 @@ def test_tracestate_w3c_p_inject(self, test_agent, test_library):
             # FIXME: nodejs paramerric app sets span.span_id to a string, convert this to an int
             assert "p:{:016x}".format(int(span.span_id)) in tracestate
 
+    # W3C Phase 3 to try adding the tag if the span id matches regardless of headers order(if tracecontext is accounted)
+    @missing_feature(context.library == "python", reason="Not implemented")
+    @missing_feature(context.library < "dotnet@2.51.0", reason="Not implemented")
+    @missing_feature(context.library == "php", reason="Not implemented")
+    @missing_feature(context.library == "nodejs", reason="Not implemented")
+    @missing_feature(context.library == "java", reason="Not implemented")
+    @missing_feature(context.library == "cpp", reason="Not implemented")
+    @missing_feature(context.library == "ruby", reason="Not implemented")
+    @missing_feature(context.library == "golang", reason="Not implemented")
+    def test_tracestate_w3c_p_phase_3_extract_inject(self, test_agent, test_library):
```

Comment:
This test should not assume the default propagation style is `datadog,tracecontext`. This can unexpectedly change and make this test hard to debug.

```suggestion
    @pytest.mark.parametrize("library_env", [{"DD_TRACE_PROPAGATION_STYLE": "datadog,tracecontext"}])
    def test_tracestate_w3c_p_phase_3_extract_inject(self, test_agent, test_library):
```

Reply:
Addressed at: https://github.com/DataDog/system-tests/pull/2385/commits/e121ae9c14c25c73a1f91df597cc255d03e9f685